internal/docstore/mongodocstore/mongo.go

```diff
@@ -146,30 +146,24 @@ type collection struct {
 	idFunc  func(docstore.Document) interface{}
 }
 
-// Options controls collection behaviors.
-type Options struct {
-	// The name of the document field to use for the document ID (MongoDB's _id field).
-	// If neither IDField nor IDFunc is set, then an IDField of "_id" is used.
-	IDField string
-
-	// A function that accepts a document and returns the value to be used for the
-	// document ID (MongoDB's _id field).
-	// IDFunc should return nil if the document is missing the information to
-	// construct an ID. This will cause all actions, even Create, to fail.
-	IDFunc func(docstore.Document) interface{}
+// OpenCollection opens a MongoDB collection for use with Docstore.
+// The idField argument is the name of the document field to use for the document ID
+// (MongoDB's _id field).
+func OpenCollection(mcoll *mongo.Collection, idField string) *docstore.Collection {
+	return docstore.NewCollection(newCollection(mcoll, idField, nil))
 }
 
-// TODO(shantuo): figure out how to handle this for for URLs. Suggestion: a query parameter
-// or path element that names a single field. That would at least handle the common case
-// where one document field is the primary key.
-
-// OpenCollection opens a MongoDB collection for use with Docstore.
-func OpenCollection(mcoll *mongo.Collection, opts Options) *docstore.Collection {
-	return docstore.NewCollection(newCollection(mcoll, opts))
+// OpenCollectionFunc opens a MongoDB collection for use with Docstore.
+// The idFunc argument is function that accepts a document and returns the value to
+// be used for the document ID (MongoDB's _id field). IDFunc should return nil if the
+// document is missing the information to construct an ID. This will cause all
+// actions, even Create, to fail.
+func OpenCollectionFunc(mcoll *mongo.Collection, idFunc func(docstore.Document) interface{}) *docstore.Collection {
```

Comment:
I think we should keep the options for future extensibility.

Reply:
+1, leave the Options even if they are empty.

Reply:
Done.